.github/workflows/main.yaml

```diff
@@ -0,0 +1,89 @@
+name: Quality and Build 
+
+on:
+  pull_request:
+    branches: [main]
+    types: [opened, synchronize, reopened, labeled]
+  push:
+    branches: [main]
+
+jobs:
+  lint:
+    runs-on: ubuntu-20.04
+    name: Linting
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v4
```

Comment:
Needed?

Reply:
Yes, a specific version of Python is needed for `ffi-napi` and `ref-napi` build process, also wanted to match the Dockerfile version